website/project/issue-pr-grading.rst

```diff
@@ -0,0 +1,422 @@
+- **Issue includes links to Google Docs (or other inaccesible documentation)**: While using Google Docs to sketch out some of the documentation, you should avoid posting links to those Google Docs from an issue, as it may cause confusion as to whether that document (or some future update to the wiki) is the most up-to-date version of that documentation. You should instead transfer the contents of the document to the wiki, and then link to the wiki when updating the issue.
+- **Issue was closed prematurely**: The issue was closed before it was actually done.
+- **Issue spanned two sprints**: The issue was started in one sprint, but was not completed until the next one.
+- [Major] **Issue spanned three or more sprints**: The issue was started in one sprint, but was reassigned to a new sprint more than once.
+- [Major] **The student reviewer did not provide any feedback and simply rubberstamped the PR.**: The reviewer for this PR approved the PR without comment. In the rare cases where there truly isn't anything to improve in the PR, make sure to leave a comment confirming that you reviewed the code and don't have any changes to suggest.
+- **The student reviewer indicated there was nothing to improve in the PR, but the senior developer identified multiple issues that the student reviewer should have been able to pick up on**: Please make sure that you do a thorough code review and, in the future, make sure to look out for the kind of issues the senior developer pointed out.
+- **The PR was created before it was ready for review (the student kept pushing commits after creating the PR, before it had been reviewed)**: When you create a PR, you are requesting a review of the code in the branch you want to merge. As such, you should not push any commits to the branch right after creating the PR; you should only do so in response to a code review. Make sure that, when you create a PR, the code is ready for review and passes all the tests.
+- [Major] **The PR contains a hodgepodge of different changes, instead of merging a branch that contains a clearly-specified and self-contained feature/change to the code.**: A branch should represent work on a concrete feature that is being added to the mainline branch, instead of a collection of many different changes.
+- [Major] **The PR is either clearly too large (1,000's of lines) or simply could've been broken up into smaller PRs, providing an opportunity to integrate more continuously.**: This PR represents a large amount of work that could've been integrated into ``dev`` through smaller PRs earlier in the quarter. Remember that you should not treat PRs as a singular event to merge all or most of your work into ``dev``. Integrating incrementally can help identify integration issues early, as well as ensure that you are not rushing to merge all your code towards the end of a sprint.
+
+Regular vs Minor issues
```

Comment:
I think it would make more sense to have this section immediately after the grading section, but this might just be personal preference. As I was reading the documentation, my first question when I saw the table was about the major vs minor distinction, so I think it might be worth addressing that immediately.

Reply:
I agree, I think it would be more relevant to include the major/minor issue description in the section after the table

Reply:
I've moved that subsection under the Grading section, and also placed some of the other existing text under a new "Delaying/Splitting Tasks" subsection.